djangocms_snippet/templates/djangocms_snippet/admin/change_form.html

```diff
@@ -3,8 +3,6 @@
 
 {% block object-tools %}
 {{ block.super }}
```

Comment:
I'm assuming that the media is being handled already somewhere else in the parent templates?

Reply:
Very djangonic: In the admin's Media class! I like the idea (which @jgadelange proposed)! It's the right place for js files to be loaded and handles static files automatically. In Django 4.2 we can even assume to have integrity parameters added for cdn downloads.

Reply:
@fsbraun Oh is it magic through the admin classes?! I don't really do that anywhere so was half expecting something like `{{ form.media }}` like you would if this was a custom form.